axis_backend/apps/authentication/views.py

```
@@ -1,3 +1,112 @@
+    'samesite': 'Lax',
+    'secure': not settings.DEBUG,  # True in production, False in development
+    'path': '/api/auth/',
+}
+
+class CookieTokenObtainPairSerializer(TokenRefreshSerializer):
+    """
+    Custom serializer for obtaining token pairs.
+    Doesn't include the refresh token in the response body.
+    """
+    @classmethod
+    def get_token(cls, user):
```

Ruff (0.14.7)

Missing return type annotation for classmethod `get_token`

(ANN206)